portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,78 @@
+body{
```

Comment:
reformat the css according to https://google.github.io/styleguide/htmlcssguide.html#CSS_Style_Rules

Reply:
Thanks

- Shortened 'introduction' to 'intro' for ID and classes in commit e3c334d and 943758c

Reply:
- Added a new line for selector decleration in commit 6563d34 ([Rule 4.2.6](https://google.github.io/styleguide/include/link.png))

Reply:
- Added Changed tab spacing to +2 and separated rules with line break ([Rule 4.2.7](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation))

Reply:
- Made selector name more generalized and purposeful in commit c33f39e ([Rule 4.1.2](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming))